### PR TITLE
Feature - Restaurant home page

### PR DESCRIPTION
**Pull request by mrnadaara**
Purpose of this PR is to implement a modular approach in code design

- Added Tab Component which can be used to switch between content
- Each front end component has been separated into smaller modules, including Tab contents
- All components are designed with Bootstrap 4

DISCLAIMER:
I've modified the linter to disable certain rules which were causing the app to break.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "linebreak-style": "off",
+      "import/prefer-default-export": "off",
+      "class-methods-use-this": "off",
+      "import/no-unresolved": "off",
+      "no-plusplus": "off"
+    }
+}
```
Comment (Oghenebrume50) [inline .eslintrc.json]: Please do not make changes to this file see this

Comment (mrnadaara) [reply .eslintrc.json]: Hey,
I've tried every possible solutions to fix the linter issues but there are some options such as no-plusplus that needs to be disabled if the code is to work. It broke my app and I'm unfortunately unwilling to enable such options

Comment (mrnadaara) [reply .eslintrc.json]: You can also see from the linter documentation that I'm allowed to make changes if necessary.
linter

Comment (mrnadaara) [reply .eslintrc.json]: I've had to make these changes because it will break my app

**Pull request merged**